lib/themes/dosomething/paraneue_dosomething/scss/patterns/_photo.scss

```diff
@@ -22,34 +26,34 @@
     img {
       width: 100%;
     }
+  }
+}
 
-    .__copy {
-      font-size: $font-smaller;
-      height: 110px;
-      line-height: 1.2;
-      overflow: hidden;
-      margin-top: ($base-spacing / 2);
-      word-wrap: break-word;
-
-      .photo__caption {
-        font-size: $font-smaller;
-      }
-
-      .footnote {
-        font-size: $font-small;
-      }
-
-      .form-actions {
-        margin-top: 6px;
-      }
-
-      @include media($large) {
-        font-size: $font-small;
-      }
-    }
+.photo__copy {
+  font-size: $font-smaller;
+  height: 110px;
+  line-height: 1.2;
+  overflow: hidden;
+  margin-top: ($base-spacing / 2);
+  word-wrap: break-word;
+
+  .photo__caption {
+    font-size: $font-smaller;
   }
 
-  .photo--actions {
-    width: 100%;
+  .footnote {
+    font-size: $font-small;
   }
+
+  .form-actions {
```

Comment:
@DFurnes I think this is no longer used right? was part of [this update](https://github.com/DoSomething/phoenix/commit/7e717b822798288fe19e0bf63d3432ebc3bd73c6#diff-b03be8fb9241bec0738afadb6c845820R43) from a year ago. I can remove it...